pkg/api/networkservice/mechanisms/vxlan/constant.go

```diff
@@ -35,10 +35,7 @@ const (
 	SrcPort = common.SrcPort
 	// DstPort - Destination vxlan listening port
 	DstPort = common.DstPort
-	// SrcOriginalIP - original src IP
-	SrcOriginalIP = common.SrcOriginalIP
-	// DstExternalIP - external destination ip
-	DstExternalIP = common.DstExternalIP
```

Comment:
The main idea that remote mechanisms don't care about original IPs.
@edwarnicke What do you think about this?

Reply:
The NSE still needs the SrcOriginalIP so it can use that to compare with its own IP to decide whether to issue an Even or Odd VNI.

Reply:
Oh... I see from below... you are basically concealing these inside methods, and so we don't need constants for them for this mechanism, correct?

Reply:
> Oh... I see from below... you are basically concealing these inside methods, and so we don't need constants for them for this mechanism, correct?

Yes, I think remote mechanisms can don't care about this kind stuff :)